### ISSUE TITLE
[Image needs adding] "Research Infrastructure Roles" chapter 

### ISSUE THREAD

**Issue by Arielle-Bennett** [the-turing-way/the-turing-way#3342]
### Summary

The chapter: https://the-turing-way.netlify.app/collaboration/research-infrastructure-roles does not have an image associated with it. One was created in a previous Book Dash but it now needs to be embedded into the guide and alt text written. 

This is linked to issue #3331 which has more detailed instructions!

### What needs to be done?

- [ ] Find the image from the Zenodo library (https://zenodo.org/records/8169292)
- [ ] Write the alt text for the image
- [ ] Open a PR to add the image to the chapter 
- [ ] PR reviewed and merged 

### Who can help?

* Anyone, this is a good first issue 

---

### Updates

*Lorem ipsum dolor sit amet, consectetur adipiscing.*

**Comment by pravintargaryen** [comment]
@Arielle-Bennett Please assign this issue to me

**Comment by Arielle-Bennett** [comment]
Hi @pravintargaryen we've opened this issue as one people might want to work on as part of this week's Book Dash - if no one addresses it today, you're welcome to open a PR which resolves this issue :)

**Comment by Arielle-Bennett** [comment]
Closing as this was addressed by #3384
